app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,20 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+/*
+ *= require font-awesome
+ */
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
```

Comment:
- [ ] It will be great if you add all of your CSS in this file. This file is in the asset pipeline and it will ensure that your CSS is well-defined in your application. At the moment the design of your application is not properly defined. Kindly reference the suggestions made by the previous reviewers